assets/i18n/pt.json

```diff
@@ -499,6 +507,7 @@
 	"I18N_START_HERE": "Clique aqui para começar!",
 	"I18N_SUBSCRIBE_BUTTON_TEXT": "Subscrever",
 	"I18N_TEACH_PAGE_HEADING": "Ajudar estudantes à volta do mundo",
+	"I18N_THANKS_PAGE_BREADCRUMB": "obrigada \nobrigado",
```

Comment:
Is the `\n` required here?

Reply:
Ah, thanks for catching that! I have fixed it and pushed again.